apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -498,45 +536,39 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
 
   // Used in levels with pre-defined "Blockly Variables"
-  blocklyWrapper.WorkspaceSvg.prototype.registerGlobalVariables = function (
-    variableList
-  ) {
+  extendedWorkspaceSvg.registerGlobalVariables = function (variableList) {
     this.globalVariables = variableList;
     this.getVariableMap().addVariables(variableList);
   };
 
-  blocklyWrapper.WorkspaceSvg.prototype.getContainer = function () {
+  extendedWorkspaceSvg.getContainer = function () {
     return this.svgGroup_.parentNode;
   };
 
-  const googleBlocklyBlocklyResize =
```

Comment:
[discussed in slack](https://codedotorg.slack.com/archives/C05DK21DAHK/p1708117535064739), `this.flyout_` is not a valid field anymore, and flyouts do not have a `resize` method, so this was not doing anything.